src/main/java/me/ywoo/domain/Price.java

```diff
@@ -0,0 +1,44 @@
+package me.ywoo.domain;
+
+import java.util.Objects;
+
+public class Price {
+    private static final int PRICE_UNIT = 1000;
+    private static final int REMAIN_IS_ZERO = 0;
+    private static final int UPPER_BOUNDARY_OF_PRICE = 100000;
+    private static final int LOWER_BOUNDARY_OF_PRICE = 1000;
```

Comment:
1000단위마다 _를 활용하면 가독성을 높일 수 있습니다.

```suggestion
    private static final int UPPER_BOUNDARY_OF_PRICE = 100_000;
    private static final int LOWER_BOUNDARY_OF_PRICE = 1_000;
```

Reply:
생각치 못했어요!! 수정하겠습니다 👍👍